modules/apps/depot/depot-test/src/testFunctional/macros/Depot.macro

```diff
@@ -71,6 +71,25 @@ definition {
 		}
 	}
 
+	macro deleteContentViaBulkEditing {
+		if (isSet(recycleBinDisabled)) {
+			ClickNoError(locator1 = "Icon#DELETE");
+
+			AssertConfirm(value1 = "Are you sure you want to delete the selected entries? They will be deleted immediately.");
+
+			Alert.viewSuccessMessage();
+		}
+		else {
+			ClickNoError(locator1 = "Icon#DELETE");
```

Comment:
@Jeremy-Ch  We should keep the trash and delete icons separate. Please create an Icon#TRASH locator and apply that to locators which have a trash icon in a separate LRQA ticket.